lxd/rsync/rsync.go

```diff
@@ -68,6 +68,10 @@ func LocalCopy(source string, dest string, bwlimit string, xattrs bool, rsyncArg
 		return "", err
 	}
 
+	if !shared.IsDir(source) {
+		return "", fmt.Errorf("rsync source (%s) is not a directory", source)
```

Comment:
We tend to use %q for quoting parameters in errors rather than "(%s)".

Also we tend to start errors with upper case characters, im not sure "Rsync" works here, perhaps just "Source %q is not a directory" (the rsync part could be added by the caller, i.e `"Failed to rsync: %w", err`, with err being `Source %q is not a directory`.